libmscore/staff.cpp

```diff
@@ -665,8 +665,8 @@ void Staff::write(XmlWriter& xml) const
             int b = i->bracketSpan();
             int c = i->column();
             if (a != BracketType::NO_BRACKET || b > 0)
-                  xml.tagE(QString("bracket type=\"%1\" span=\"%2\" col=\"%3\"").arg(static_cast<int>(a)).arg(b).arg(c));
-            }
+                   xml.tagE(QString("bracket type=\"%1\" span=\"%2\" col=\"%3\"").arg(static_cast<int>(a)).arg(b).arg(c));
+           }
```

Comment:
Why those formatting changes? It was properly indented before

Reply:
I didn't follow the recommended git workflow and then things went downhill when I wanted to bring my local repository up to date. I will not make the same mistake again.